src/go_to_class/wdGoToClass/$_getLibrary.omh

```diff
@@ -1 +1 @@
-Quit method $libs.[ilLibraries.Name]
\ No newline at end of file
+Quit method $libs.[pick(isclear($cclass.$userinfo),$cclass.$userinfo,ilLibraries.Name)]
```

Comment:
I would keep this line the same since we're getting the library that's currently selected, which is stored in `ilLibraries`.